[Ready] Add api reference (#1)
barmej/react-native-youtube-player: README.md

```

## API reference
|   Property    |      Type     |  description  |
| :---          | :---:          |:---          |
| videoId(required)| string     | Youtube video Id  |
| autoPlay      |  Boolean      | Auto play the video |
  onError       |  () => void   |Execute a function on error     |

> | loop(required)| Boolean   | Loop the video|

Reviewer: `loop` not required 😄

Author: 😅

```

## API reference
|   Property    |      Type     |  description  |
| :---          | :---:          |:---          |
| videoId(required)| string     | youtube video Id  |
| autoPlay      |  Boolean      | auto play the video |
  onError       |  function() => void   |execute a function on error     |
| loop(required)| Boolean   | loop the video|
| style         | object        | You can pass this to override some default styles |

> | topBar        | function(play: boolean, fullScreen: boolean)  | function which takes an object with the play and fullScreen status and return a react element to be used as a topBar   |

Reviewer: ```suggestion
| topBar        | (play: boolean, fullScreen: boolean) => React.ReactNode  | function which takes  the play and fullScreen status and return a react element to be used as a topBar   |
```

| showFullScreenButton| Boolean |   display a button to allow user to see the video on fullScreen     |
| onFullScreen  | function(fullScreen: Boolean) | execute a function in fullScreen |

> | onStart       | function() => void    | execute a function on start |

Reviewer: ```suggestion
| onStart       | () => void    | execute a function on start |
```

Reviewer: same for others
